### ISSUE TITLE
Unicode normalization for bert-cased models

### ISSUE THREAD

**Issue by alexeyr** [huggingface/transformers#5738]
# 🐛 Bug

## Information

Model I am using (Bert, XLNet ...): Bert

Language I am using the model on (English, Chinese ...): English

The problem arises when using:
* [ ] the official example scripts: (give details below)
* [x] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x] my own task or dataset: (give details below)

## To reproduce

Tokenize text containing combining marks with `BertTokenizer`. E.g.
```
In [1]: from transformers.tokenization_bert import *                                                                                                     

In [2]: tokenizer = BertTokenizer.from_pretrained("bert-base-cased")                                                                                     

In [3]: tokenizer.tokenize("\u00E1")                                                                                                                     
Out[3]: ['á']

In [4]: tokenizer.tokenize("a\u0301")                                                                                                                    
Out[4]: ['a', '##́']

In [5]: tokenizer.tokenize("a\u0300")                                                                                                                    
Out[5]: ['[UNK]']

In [6]: tokenizer.tokenize("\u00E0")                                                                                                                     
Out[6]: ['à']
```

Results for `BertTokenizerFast` are the same.

## Expected behavior

`"\u00E1"` and `"a\u0301"` should ideally be tokenized the same way; so should `"\u00E0"` and `"a\u0300"`. If existing behavior should be preserved, maybe add an optional argument.

## Environment info

     
- `transformers` version: 3.0.2
- Platform: Linux-4.4.0-18362-Microsoft-x86_64-with-Ubuntu-18.04-bionic
- Python version: 3.6.9
- PyTorch version (GPU?): 1.4.0 (False)
- Tensorflow version (GPU?): not installed (NA)
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No
